Lecture2/Task5/Task5L2.java

```diff
@@ -0,0 +1,154 @@
+import java.io.*;
+import java.util.*;
+
+public class Task5L2 {
+
+    private static final String REGEX_NAME_LASTNAME = "[a-zA-Z]+";
+    private static final String MESSAGE_AGAIN = "Again? Really? Try something else :)";
+
+    private static BufferedReader reader = new BufferedReader ( new InputStreamReader ( System.in ) );
+
+    public static
+    void main ( String[] args ) throws Exception {
+        System.out.println ( "Let's start the game!!!" );
+        Console console = System.console ( );
+        String name = null;
+        while (name == null)
+            name = getWord ( console );
+        String lastname = null;
+        while (lastname == null)
+            lastname = getWord ( console );
+        System.out.println ( "\nNow try to solve secret name and lastname ;)" );
+        List <String> listOfWords = new ArrayList <> ( );
+        List <String> listOfLetters = new ArrayList <> ( );
+        char[] nameArray = name.toCharArray ( );
+        char[] lastnameArray = lastname.toCharArray ( );
+        char[] starNameArray = new char[ name.length ( ) ];
+        for ( int i = 0 ; i < starNameArray.length ; i++ )
+            starNameArray[ i ] = '*';
+        char[] starLastnameArray = new char[ lastname.length ( ) ];
+        for ( int i = 0 ; i < starLastnameArray.length ; i++ )
+            starLastnameArray[ i ] = '*';
+        boolean checkResultName = false;
```

Comment:
boolean names could be more accurate, like isFNameOpened, isLNameOpened

Reply:
fixed